nixos/maintainers/scripts/cloudstack/cloudstack-image.nix

```diff
@@ -0,0 +1,24 @@
+# nix-build '<nixpkgs/nixos>' -A config.system.build.cloudstackImage --arg configuration "{ imports = [ ./nixos/maintainers/scripts/cloudstack/cloudstack-image.nix ]; }"
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  imports =
+    [ ../../../modules/virtualisation/cloudstack-config.nix ];
+
+  system.build.cloudstackImage = import ../../../lib/make-disk-image.nix {
+    inherit lib config;
+    pkgs = import ../../../.. { inherit (pkgs) system; }; # ensure we use the regular qemu-kvm package
```

Comment:
Why not just use `pkgs` here? What do you mean with regular qemu-kvm package?

Reply:
I copied this (comment included) from the `nova-image.nix` in `../openstack`. This was introduced in PR #23168 without much details about this point. It may have been stolen from `ec2/amazon-image.nix`. For EC2, this has been introduced in https://github.com/NixOS/nixpkgs/commit/0d3738cdcc48bd32a7eae9b914440b67c8db252e.

Reply:
This forcing a re-import of nixpkgs sounds wrong, but… cc @nlewo? (will let you poke eelco if you just took the line from his commit)

Reply:
Can you test if using `pkgs` just work?

Reply:
@Ekleog sorry, I can't help on this... I think, I just copied this line:/ As suggested by @Mic92, we could try with `pkgs`.

Reply:
It works just with `pkgs`. I'll update the PR.